codegen/winterfell/src/air/boundary_constraints.rs

```diff
@@ -1,17 +1,18 @@
 use core::panic;
 
-use super::{
-    AirIR, AlgebraicGraph, Codegen, ConstraintDomain, ElemType, Impl, NodeIndex, Operation,
-    TraceAccess, Value,
+use air_ir::{
+    Air, AlgebraicGraph, ConstraintDomain, NodeIndex, Operation, TraceAccess, TraceSegmentId, Value,
 };
 
+use super::{Codegen, ElemType, Impl};
+
 // HELPERS TO GENERATE THE WINTERFELL BOUNDARY CONSTRAINT METHODS
 // ================================================================================================
 
 /// Adds an implementation of the "get_assertions" method to the referenced Air implementation
-/// based on the data in the provided AirIR.
+/// based on the data in the provided Air.
```

Comment:
nit: I think this actually becomes a bit confusing now, because we're using "Air" to talk about two different things (the Air struct implementation that is being output, and the IR that is being referenced). Can we add a qualifier to one of them? Same comment with `add_fn_get_aux_assertions` below